Bio/FSSP/FSSPTools.py

```diff
@@ -72,9 +76,9 @@ def filter(sum_dict, align_dict, filter_attribute, low_bound, high_bound):
     """
     new_sum_dict = FSSP.FSSPSumDict()
     new_align_dict = copy.deepcopy(align_dict)
-#    for i in align_dict:
-#        new_align_dict[i]  = copy.copy(align_dict[i])
-#    new_align_dict = copy.copy(align_dict)
+    #    for i in align_dict:
+    #        new_align_dict[i]  = copy.copy(align_dict[i])
+    #    new_align_dict = copy.copy(align_dict)
```

Comment:
The original code should have had the ``#`` indented to match the code, the net result is post black this looks silly.

That said, with the use of deepcopy I think the old commented out code can just be removed now?

Reply:
Agreed, it was commented on 2001, I have removed it.